pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -3603,3 +3709,16 @@ func lookupCapabilities(ctx *domainContext) (*types.Capabilities, error) {
 	}
 	return &capabilities, nil
 }
+
+func (ctx *domainContext) retrieveNodeNameAndUUID() error {
```

Comment:
This is our own nodes nodename, right?
Can't it retrieve that at domainmgr startup?

And why is "UUID" part of the name of the function?

Reply:
This part I used to at the startup of domainmgr, the 'waitEdgeNodeInfo' only wait for max of 5 minutes, so i could not relied on this. In previous review of this PR, you pointed out this, and I removed it. So, good point, I should also remove this function here.
I used to have UUID in the function() in previous code before this PR, will remove that.